docs/getting-started.md

```diff
@@ -299,8 +299,8 @@ account, for developing your own study:
        system tray docker icon; select "settings"; select "shared drives".  This
        setting does not exist in the *WSL 2* backend.
     1. Test Docker and opensafely work together. Open an Anaconda Prompt, and run
-       `opensafely pull cohortextractor`. This will pull down the OpenSAFELY
-       cohortextractor images, which can be used to run actions in your study.  The
+       `opensafely pull ehrql`. This will pull down the OpenSAFELY
+       ehrql images, which can be used to run actions in your study.  The
```

Comment:
thought: Unrelated to this PR, is "can be used to run actions in your study" correct here? It doesn't sound quite right to me, but could be my reading.